target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -193,7 +193,7 @@ glinet,gl-mt750)
 	;;
 glinet,gl-mt300n-v2)
 	set_wifi_led "$boardname:red:wlan"
-	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x1"
+	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
```

Comment:
This change has nothing to do with current commit. You could either use a separated commit for this or drop this change.